api/api/specification/suggestions.yaml

```diff
@@ -173,3 +173,29 @@ paths:
       responses:
         '204':
           description: List of deleted tags
+  /suggestions/{suggestion_id}/assign/{user_id}:
+    parameters:
+      - name: suggestion_id
+        in: path
+        type: integer
+        required: true
+      - name: user_id
+        in: path
+        type: integer
+        required: true
+    put:
+      operationId: api.logic.suggestions.assign_to_user
+      responses:
+        "200":
```

Comment:
https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete pitäiskö tuo olla ennemmin 202?